framework/src/modules/token/method.ts

```diff
@@ -310,6 +310,12 @@ export class TokenMethod extends BaseMethod {
 			throw new Error(`TokenID ${tokenID.toString('hex')} is not native token.`);
```

Comment:
Thanks :pray: for pointing out the missing parts;
The commits cover the pointed out missing implementations;

- [Zero amount check for mint](https://github.com/LiskHQ/lisk-sdk/pull/8247/commits/c0cce36c5f3b8da8a178e49fde03b219c6dc71c6)
- [Zero amout check for burn](https://github.com/LiskHQ/lisk-sdk/pull/8247/commits/c7f54e3cf2f16942c15348ad369111f5ab93e4dc)
- [transferCrossChain throws if receiving chain is own chain](https://github.com/LiskHQ/lisk-sdk/pull/8247/commits/1ea809bda46504e0271d668727ab14b34aa2d34d)
- [getEscrowedAmount throws if escrow chain is own chain ](https://github.com/LiskHQ/lisk-sdk/pull/8247/commits/f3403c6fb1d02626fbdbf0eef30de6921aba9b74)
- [getEscrowedAmount returns 0 if escrowAccount does not exist, instead of throwing error ](https://github.com/LiskHQ/lisk-sdk/pull/8247/commits/3fd078d3dbc8cfd134f00b6eb7bcfbd75b0e5523)